__tests__/validations/component.test.tsx

```diff
@@ -890,3 +894,68 @@ test("throws an error when a reserved keyword is used 'WRAPPER'", (t: Context):
 
   t.throws(() => validateComponents(components, []));
 });
+
+test('Throw when one of the prefabs configuration options is invalid', (t: Context): void => {
+  const prefabs = [
+    {
+      name: 'Component Name',
+      icon: 'TitleIcon',
+      category: 'CONTENT',
+      structure: [
+        {
+          name: 'something',
+          options: [
+            {
+              value: '',
+              label: 'something',
+              key: 'something',
+              type: 'PROPERTY',
+              configuration: {
+                allowedKinds: 'TEXT',
+              },
+            },
+          ],
+          descendants: [],
+        },
+      ],
+    },
+  ] as unknown as Prefab[];
+
+  t.throws(() => validatePrefabs(prefabs, {}));
+});
+
+test('Success when the reconfigure configuration optioms of the prefabs are valid', (t: Context): void => {
+  const prefabs = [
```

Comment:
Can't you do `const prefabs: Prefab[] = [`
Instead of assigning the type on line 958